### PR TITLE
make kompose go get-able

### LINKED ISSUE #216: Make go get'able (or at least go installable)
It's odd that kompose's main package is named `main` rather than `kompose` like most Go projects Additionally, since there's no top level package it seems like it'd be reasonable to enable `go get github.com/kubernetes-incubator/kompose` to work.

Additionally there's a dependence on `-tags experimental` due to a docker dependency[0]. Removing that import here[1], (say by copying the structs) would also break the dependency on github.com/docker/docker and remove 19 direct imports from vendor.

```
$ cat Godeps/Godeps.json | jq .Deps[].ImportPath | grep '"github.com/docker/docker'
"github.com/docker/docker/api/types"
"github.com/docker/docker/api/types/blkiodev"
"github.com/docker/docker/api/types/container"
"github.com/docker/docker/api/types/filters"
"github.com/docker/docker/api/types/mount"
"github.com/docker/docker/api/types/network"
"github.com/docker/docker/api/types/registry"
"github.com/docker/docker/api/types/strslice"
"github.com/docker/docker/api/types/swarm"
"github.com/docker/docker/api/types/versions"
"github.com/docker/docker/cli/command/bundlefile"
"github.com/docker/docker/opts"
"github.com/docker/docker/pkg/mount"
"github.com/docker/docker/pkg/signal"
"github.com/docker/docker/pkg/system"
"github.com/docker/docker/pkg/term"
"github.com/docker/docker/pkg/term/windows"
"github.com/docker/docker/pkg/urlutil"
"github.com/docker/docker/runconfig/opts"
```

[0] https://github.com/docker/docker/blob/15ea28f6db7339f8a44078f3ef70484c96eebce7/cli/command/bundlefile/bundlefile.go
[1] https://github.com/kubernetes-incubator/kompose/blob/d9899b788d0e2051221e1b993f5a48268f9f3758/pkg/loader/bundle/bundle.go#L26

### PR DESCRIPTION
**Pull request by ngtuna**
Fix #216

### CODE REVIEW

**Comment by ericchiang** [comment]
I was implying the CLI be go-getable in #216

**Comment by ngtuna** [comment]
@ericchiang  you mean `go get github.com/kubernetes-incubator/kompose/cli` ?

**Comment by ericchiang** [comment]
@ngtuna I was implying renaming the package `main` to `kompose` so go get would install the binary correctly. For users familiar with Go this is what they expect for most CLI go projects.

So at minimum

```
go get github.com/kubernetes-incubator/kompose/cli/kompose
```

**Comment by ericchiang** [comment]
It's obviously not critical. But it's pretty standard for Go projects :)

**Comment by ngtuna** [comment]
Okay thanks @ericchiang for pointing it out. It's pretty easy to rename the package. I will consider more on removing experimental tag of bundlefile.

**Comment by ngtuna** [comment]
@ericchiang I see it would be better to move main/main.go to the parent directory but not sure if it violates the standard of kubenertes-incubator or not. That is a bit of weird in path of the package when doing `go get github.com/kubernetes-incubator/kompose/cli/kompose`. It would be better with:

```console
$ go get github.com/kubernetes-incubator/kompose
```

**Comment by ericchiang** [comment]
> $ go get github.com/kubernetes-incubator/kompose

I'm for this (but that's just a preference).

**Comment by dustymabe** [comment]
Works for me but I can't officially review so it's not worth much.

**Comment by ngtuna** [comment]
ping @ericchiang @kadel Could you add your review on this ?

**Comment by ericchiang** [comment]
lgtm but I'll let @kadel add the review.

File: README.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ tar -xvf kompose_linux-amd64.tar.gz --strip 1
 sudo mv kompose /usr/local/bin
 ```
 
+Otherwise, you could take the latest development package from master branch. Make sure your PATH includes the $GOPATH/bin directory so your commands can be easily used:
```
Comment (janetkuo) [inline README.md]: s/Otherwise/Alternatively/g

**Comment by janetkuo** [comment]
I found that `make binary` doesn't work on HEAD (not broken by this PR)

```console 
$ make binary
CGO_ENABLED=1 ./script/make.sh binary
---> Making bundle: binary (in .)
# command-line-arguments
./main.go:35: cannot use command.BeforeApp (type func(*"github.com/kubernetes-incubator/kompose/vendor/github.com/urfave/cli".Context) error) as type func(*"github.com/skippbox/kompose/vendor/github.com/urfave/cli".Context) error in assignment
./main.go:36: cannot use append(command.CommonFlags()) (type []"github.com/kubernetes-incubator/kompose/vendor/github.com/urfave/cli".Flag) as type []"github.com/skippbox/kompose/vendor/github.com/urfave/cli".Flag in assignment
./main.go:40: cannot use command.ConvertCommandDummy() (type "github.com/kubernetes-incubator/kompose/vendor/github.com/urfave/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/urfave/cli".Command in array or slice literal
./main.go:44: cannot use command.UpCommand() (type "github.com/kubernetes-incubator/kompose/vendor/github.com/urfave/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/urfave/cli".Command in array or slice literal
./main.go:45: cannot use command.DownCommand() (type "github.com/kubernetes-incubator/kompose/vendor/github.com/urfave/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/urfave/cli".Command in array or slice literal
make: *** [binary] Error 2
```

File: README.md (hunk 1 above)
Comment (ngtuna) [inline README.md]: Committed amend

**Comment by ngtuna** [comment]
> I found that make binary doesn't work on HEAD (not broken by this PR)

Could you re-run it ? I just run `make binary` successfully on HEAD.

**Comment by janetkuo** [comment]
>> I found that make binary doesn't work on HEAD (not broken by this PR)

> Could you re-run it ? I just run make binary successfully on HEAD.

From the error log it seems that it's because I'm still using skippbox/kompose/
```
./main.go:35: cannot use command.BeforeApp (type func(*"github.com/kubernetes-incubator/kompose/vendor/github.com/urfave/cli".Context) error) as type func(*"github.com/skippbox/kompose/vendor/github.com/urfave/cli".Context) error in assignment
```

`make binary` works after I change my local directory from skippbox/kompose to kubernetes-incubator/kompose.

**Comment by ngtuna** [comment]
Merging

**Pull request merged**